connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -86,10 +86,11 @@ public ConnectorsResource(Herder herder) {
     @POST
     @Path("/")
     public Response createConnector(final @QueryParam("forward") Boolean forward,
-                                    final CreateConnectorRequest createRequest) throws Throwable {
+                                     final CreateConnectorRequest createRequest) throws Throwable {
         String name = createRequest.name();
-        if (name.contains("/")) {
-            throw new BadRequestException("connector name should not contain '/'");
+        if (name == null) {
+            // the request did not contain a connector name, reject it as a bad request
+            throw new BadRequestException("connector name is mandatory in create connector requests");
```

Comment:
What about "connector name must be supplied"?

Reply:
Will change to be consistent with the other message above.